templates/layout.html.twig

```diff
@@ -134,19 +134,19 @@
                         <li class="divider"></li>
                     {% endif %}
                     <li>
-                        <a href="#" class="js-theme-toggle" data-theme="light">
+                        <a href="#" data-action="click->dark-theme#useLight:prevent">
```

Comment:
`:prevent` is used to not navigate to `#`

Reply:
technically those should be buttons, not links, but let's keep that for a future improvement (there are way more places to update..)